src/components/LoanCards/AlgoliaLoanCardAdapter.vue

```diff
@@ -72,7 +72,7 @@ export default {
 					default: defaultImage,
 					retina: retinaImage
 				},
-				lenderRepaymentTerm: _get(this.loan, 'lenderRepaymentTerm').toString(),
+				lenderRepaymentTerm: parseInt(_get(this.loan, 'lenderRepaymentTerm'), 10),
```

Comment:
I think the data must have changed on the backend push to Algolia to make this a number. So, now we ensure it's a number.